gnd/src/main/java/com/google/android/gnd/service/RemoteDatastore.java

```diff
@@ -27,9 +27,10 @@
 import java.util.List;
 
 /**
- * Data service is treated as if it's remote, though implementations may cache data locally as well.
+ * Defines API for accessing data in a remote datastore. The store is treated as if it's remote,
+ * though implementations may cache data locally as well.
  */
-public interface RemoteDataService {
+public interface RemoteDatastore {
```

Comment:
Super-nit: I'd expect this to be `RemoteDataStore` -- to me, "Datastore" is a database product in Google Cloud Platform, "data store" is the generic concept of a store for data.

I definitely don't feel strongly enough about this to push very hard if you push back, though.

Reply:
Ah, that's right. Fixing this now.

Reply:
I've also renamed other instances of Datastore throughout.